### PR TITLE
Add xarray benchmark

### PR DESCRIPTION
**Pull request by quasiben**
PR adds xarray to benchmark.  Also, we limit the number of threads to 1 per workers

### CODE REVIEW

**Review by jakirkham — APPROVED** [review]
LGTM. Thanks for adding this Ben! 😄

File: nightly-benchmark/dgx-benchmark-historic-runs.csv [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+20201129,simple,0.5745548963546753,0.05723552947310567
+20201129,shuffle,24.421786642074586,1.6292441450772235
+20201129,rand_access,0.007096219062805176,0.0028575719385972847
+20201130,simple,0.5799180269241333,0.0595338902855331
+20201130,shuffle,24.354144883155822,0.46329367607976524
+20201130,rand_access,0.010492658615112305,0.001179463979499339
+20201130,simple,0.5958759546279907,0.02384114128906203
+20201130,shuffle,22.320822596549988,0.9106601352938861
+20201130,rand_access,0.010452866554260254,0.0008015916163072284
+20201130,anon_mean,102.93944501876831,3.4379303100271668
```
Comment (jrbourbeau) [inline nightly-benchmark/dgx-benchmark-historic-runs.csv:25]:
```suggestion
20201130,anom_mean,102.93944501876831,3.4379303100271668
```

File: nightly-benchmark/publish_benchmark.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 g = Github(os.environ["GITHUB_ACCESS_TOKEN"])
 
-operations = ['simple', 'shuffle', 'rand-access']
+operations = ['simple', 'shuffle', 'rand-access', 'anon-mean']
```
Comment (jrbourbeau) [inline nightly-benchmark/publish_benchmark.py:8]:
```suggestion
operations = ['simple', 'shuffle', 'rand-access', 'anom-mean']
```

File: nightly-benchmark/publish_benchmark.py [hunk 3]
```diff
@@ -62,6 +62,9 @@
   Profile](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-rand-access-scheduler.html)
 - [Simple
   Profile](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-simple-scheduler.html)
+- [Anon
+  Mean](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-anon-mean-scheduler.html)
+
```
Comment (jrbourbeau) [inline nightly-benchmark/publish_benchmark.py:67]:
```suggestion
- [Anom
  Mean](https://raw.githack.com/quasiben/dask-scheduler-performance/benchmark-images/assets/{today}-anom-mean-scheduler.html)

```

**Review by jrbourbeau — COMMENTED** [review]
Nitpick: I _think_ we want `anom` instead of `anon`

**Comment by jrbourbeau** [comment]
Woops, I was too slow. I'll open a separate PR with these suggestions

**Comment by quasiben** [comment]
@jrbourbeau, sorry!!!!  I can take care of that

**Pull request merged**